.eslintrc.json

```diff
@@ -1,6 +1,7 @@
 {
   "env": {
-    "es6": true,
+    "es2017": true,
```

Comment:
Why do we need this?

Reply:
Discussed offline. This was specifically added to support async function () syntax in the before hook for each suite of tests, otherwise eslint would throw an error.